### PR TITLE
Feature/top nav bar

### PR DESCRIPTION
**Pull request by NIanBeIx**
### Changes
- all project view with card containers

### CODE REVIEW

**Review by NIanBeIx — COMMENTED** [review]
I could not import images in public folder, so I temporarily stored the images I need under src/static/images

File: src/Components/ProjectPage/ProjectCardContent.js [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react';
+import CardMedia from '@material-ui/core/CardMedia';
+import CardContent from '@material-ui/core/CardContent';
+import McLogo from '../../static/images/McDonaldLogo.png';
+import { Typography } from '@material-ui/core';
+
+export default class FloatingAddButton extends Component {
```
Comment (eugenechia95) [inline src/Components/ProjectPage/ProjectCardContent.js]: Wrong class name  "FloatingAddButton" ??? :sweat_smile:

File: src/Components/ProjectPage/ProjectCard.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+  const handlePopoverOpen = () => {
+    setDisplay(true);
+  };
+
+  const handlePopoverClose = () => {
+    setDisplay(false);
+  };
+
+  const handelClick = () => {
```
Comment (eugenechia95) [inline src/Components/ProjectPage/ProjectCard.js]: typo "handelClick" :sweat_smile:

File: src/Components/ProjectPage/ProjectCard.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+    window.location.href = `${match.path}/${props.projectId}`;
+  };
+
+  return (
+    <div onMouseEnter={handlePopoverOpen} onMouseLeave={handlePopoverClose}>
+      <Card className={classes.root}>
+        <CardActionArea onClick={handelClick}>
```
Comment (eugenechia95) [inline src/Components/ProjectPage/ProjectCard.js]: typo "handelClick" :sweat_smile:

**Review by eugenechia95 — COMMENTED** [review]
Is this still a WIP??? Or maybe i was overzealous about reviewing lol :sweat_smile: 

- Think there's a minor bug whereby if i hover over too quickly, this happens (Can take note for now, not urgent change):
image

- FloatingActionButton cannot be viewed. What is it for??? :thinking: 

- > I could not import images in public folder, so I temporarily stored the images I need under src/static/images

    Images can be viewed, not an issue

**Comment by NIanBeIx** [comment]
> Is this still a WIP??? Or maybe i was overzealous about reviewing lol 😅
> 
> * Think there's a minor bug whereby if i hover over too quickly, this happens (Can take note for now, not urgent change):
>   image
> * FloatingActionButton cannot be viewed. What is it for??? 🤔
> * > I could not import images in public folder, so I temporarily stored the images I need under src/static/images
>   
>   
>   Images can be viewed, not an issue

**Comment by NIanBeIx** [comment]
-- The views and layout are basic containers, the details inside (like project details page and project creation form content) could be changed.
-- The floating button is to display the creation form (added now)
-- The model creation form and project creation form's content will be refined after the details are provided (by modeling team)
-- The image on the card can be changed by an uploading field in the creation form (will be implemented later, need to set up cloud storage)

**Comment by eugenechia95** [comment]
Apart from the typos, everything else lgtm :+1:

**Review by eugenechia95 — APPROVED** [review]

**Pull request merged**